### DISCUSSION TITLE
Docs on Plugins

### DISCUSSION THREAD

**Discussion by Purus** [viames/pair#1] [General]
I am interested in this project as I am looking for a pluggable project where plugins can be installed with ease. Can you point me to any docs which explains about plugin management? Thanks.

**Comment by viames** [comment]
Hi Purus, I'm not sure if the Pair boilerplate which contains the form manager for loading has been updated to the current Pair framework version.
Having been used mostly for private, boilerplate-independent applications, it probably needs an overhaul.

However, the plugin system allows you to install via a ZIP file that contains the manifest.xml file everything that will be enclosed in the folder of the path /modules.
The same operation applies to template-type plugins.
Through the boilerplate there are two modules that allow you to dynamically create and download the ZIP of each module and template that make up the boilerplate itself.
With the same logic, you can create a plugin of a custom type, for example "socials".

**Comment by Purus** [comment]
Thanks for the info. I will have a look. Good job on the framework.
